reflectable/bin/reflectable_builder.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2016, the Dart Team. All rights reserved. Use of this
+// source code is governed by a BSD-style license that can be found in
+// the LICENSE file.
+
+/// Performs the reflectable code generation as a 'package:build' builder.
+///
+/// Expects the package name as the first argument, followed by the entry
+/// points to transform.
+
+library reflectable.bin.reflectable_builder;
+
+import 'package:barback/src/transformer/barback_settings.dart';
+import 'package:build_barback/build_barback.dart';
+import 'package:build_runner/build_runner.dart';
+import 'package:reflectable/transformer.dart';
+
+main(List<String> arguments) async {
```

Comment:
Note that you could get the root package this way:

```dart
var graph = new PackageGraph.forThisPackage();
var package = graph.root.name;
```

Reply:
Great, I've been looking for that! Done.